test/remote/gateways/remote_stripe_emv_test.rb

```diff
@@ -143,4 +143,9 @@ def test_authorization_and_void_with_emv_contactless_credit_card
     assert_success void
   end
 
+  def test_authorization_emv_credit_card_in_us_with_metadata
+    @gateway = StripeGateway.new(fixtures(:stripe_emv_us))
+    assert authorization = @gateway.authorize(@amount, @emv_credit_cards[:us], @options.merge({:metadata => {:this_is_a_random_key_name => 'with a random value', :i_made_up_this_key_too => 'canyoutell'}, :order_id => "42", :email => "foo@wonderfullyfakedomain.com"}))
```

Comment:
I'm considering registering `wonderfullyfakedomain.com` just to mess with this...
Just to confirm, it will not send an email, right?